lib/streams.gd

```diff
@@ -726,14 +726,16 @@ DeclareOperation( "OutputTextString", [ IsList, IsBool ] );
 ##  <#GAPDoc Label="OutputTextFile">
 ##  <ManSection>
 ##  <Oper Name="OutputTextFile" Arg='filename, append'/>
+##  <Oper Name="OutputGzipFile" Arg='filename, append'/>
 ##
 ##  <Description>
 ##  <C>OutputTextFile( <A>filename</A>, <A>append</A> )</C> returns an output stream in the
 ##  category <C>IsOutputTextFile</C> that writes received characters to the file
 ##  <A>filename</A>.  If <A>append</A> is <K>false</K>, then the file is emptied first,
 ##  otherwise received characters are added at the end of the file.
 ##  If <A>filename</A> ends in <C>.gz</C> then the file will be
```

Comment:
Sorry, your message is too brief for me to guess the problem?

My aim is now OutputTextFile has it's 4.10 behaviour -- it never gzips, and you use OutputGzipFile explicitly if you want to output with gzip compression.

Reply:
I see the problem, yes, docs are still wrong.

Reply:
Also, I noticed I was still compressing things like `OpenLog`. Maybe we want that, but for now let's do the minimal.

The only thing I'm now compressing if it ends .gz without prompting is SaveWorkspace, because it's just a binary file most people wouldn't understand anyway, and it compresses really well.